src/core_ocean/analysis_members/Registry_lagrangian_particle_tracking.xml

```diff
@@ -307,6 +315,13 @@
 		<var name="transfered" type ="integer" dimensions="nParticles Time" units="unitless"
 			 description="flag to monitor if the particle was transfered"
 		/>
+<!-- TEMP FIELDS THAT NEED MOVED TO NonHalo below after verifying interpolation is correct -->
+		<var name="temperature" type="real" dimensions="nParticles Time" units="kg m^{-3}"
```

Comment:
@pwolfram -- should temperature units be "degC" or "K"?

Reply:
It should be, thanks for catching this typo.  It should also have in its description that it is potential temperature, not just regular temperature.